### DISCUSSION TITLE
How to classify video?

### DISCUSSION THREAD

**Discussion by Vista1nik** [infinitered/nsfwjs#714] [Questions]
Hello. I have a video uploader. 
I want to classify more than first video frame. Is it possible?

Thanks.

**Comment by tsehori** [comment]
Hey @Vista1nik, just throwing some idea in the air, might it be possible to convert the video to a gif and then use the module? This might take a long time though (converting + classifying).
Another idea might be taking some of the frames (every second?) and checking them. Clearly not perfect

**Comment by GantMan** [comment]
Take a look at the code I started in ticket #39 .  I'm still hoping someone will take this on:

https://github.com/infinitered/nsfwjs/issues/39

**Comment by jinsel** [comment]
Or can it be like first convert video to image, then image to Gif? Because direct Video to Gif maybe a little tough than the image to Gif

**Comment by seachenjy** [comment]
you can cover video to image. like ffmpeg4 or canvas
